cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6526,7 +6526,7 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = index=* sourcetype!="WinRegistry" NOT (sourcetype="WinEventLog" user_type="computer") NOT (sourcetype="meraki:accesspoints" (NOT eventData.identity="*")) (user=* OR User=* OR UserName=* OR Username=* OR userName=* OR username=* OR user_name=* OR src_user=*) \
+search = index=* sourcetype!="WinRegistry" NOT (sourcetype="WinEventLog" user_type="computer") NOT (sourcetype="meraki:accesspoints" (NOT eventData.identity="*")) NOT (action=fail*) (user=* OR User=* OR UserName=* OR Username=* OR userName=* OR username=* OR user_name=* OR src_user=*) \
```

Comment:
Let's exclude some of the big sources that we are aware of to reduce the search space.

pan:traffic, fortigate-traffic-logs (not sure exact sourcetype), etc

Reply:
there are many sourcetypes having action field. 

do we have to exclude logs for some login specific sourcetypes or for any sourcetypes?

Reply:
for performance completely exclude some of the unuseful sourcetypes